apps/elasticsearch/templates/services/app/configs/elasticsearch.yml.j2

```diff
@@ -0,0 +1,7 @@
+cluster.name: {{ elasticsearch_cluster_name }}
+network.host: "0.0.0.0"
+bootstrap.memory_lock: false
+discovery.zen.ping.unicast.hosts: elasticsearch-discovery.{{ project_name }}.svc
+discovery.zen.minimum_master_nodes: 2
+xpack.security.enabled: false
```

Comment:
It made me notice that `discovery.zen.minimum_master_nodes` is an hardcoded value instead of using `{{elasticsearch_minimum_master_nodes}}` :man_facepalming: 

I'll add more comments here, but the important values are commented in `main.yml`.